src/bundle/Resources/public/scss/_general.scss

```diff
@@ -156,3 +156,7 @@ select {
 .ez-prevent-click * {
     pointer-events: none !important;
 }
+
+.ez-is-tree-resizing {
+    user-select: none;
```

Comment:
According to https://caniuse.com/#search=user-select this will work only on chrome. We should add prefixes.

Reply:
Good catch!